jenkins/release.jenkinsfile

```diff
@@ -120,7 +120,8 @@ pipeline {
                                 string(name: 'RELEASE_VERSION', value: "${OPERATOR_VERSION}"),
                                 booleanParam(name: 'DOCKER_HUB_PROMOTE', value: true),
                                 booleanParam(name: 'ECR_PROMOTE', value: true),
-                                booleanParam(name: 'TAG_LATEST', value: true)
+                                booleanParam(name: 'TAG_LATEST', value: true),
+                                booleanParam(name: 'TAG_MAJOR_VERSION', value: false)
```

Comment:
Don't we want to update the major version tag? Example: 2.5.1 -> 2

Reply:
Adding @swoehrl-mw.
Sayali mostly the operator is installed and managed via helm and helm charts have fixed versions in `Chart.yml`, `docker run` wont work here hence I dont see a need to keep `TAG_MAJOR_VERSION` which adds more noise.

Reply:
The [docker-promotion](https://build.ci.opensearch.org/job/docker-promotion/) job has default `TAG_MAJOR_VERSION` to `true` hence disabling it.